### PR TITLE
Step1

### PR DESCRIPTION
**Pull request by wendyh26**
The pricingutil.cpp was not filled with the functions. Based on the provided formula, I wrote the functions and some unit tests including some edge case. The tests include zero interest rate, big Oleo constant, negative price, negative interest rate, etc. to test whether the function is correct. The unit test was the challenging part because I don’t know how many edge cases exists there. And yes I do have enough guidance to complete this task.

### CODE REVIEW

File: src/pricingutil.cpp [hunk 1]
```diff
@@ -1 +1,15 @@
 #include "pricingutil.h"
+
+
+PricingUtil::PricingUtil() {
+
```
Comment (minnce) [inline src/pricingutil.cpp]: initialize val here

**Review by minnce — CHANGES_REQUESTED** [review]

**Review by minnce — APPROVED** [review]
lgtm!
